scripts/release/tickets.py

```diff
@@ -0,0 +1,98 @@
+import os
+import re
+import requests
+
+from branch_diff import get_branch_diff
+
+branch_pattern = re.compile(r"^i(\d+)($|_)")
```

Comment:
Consider `"^i(\d+)($|[_-])"` - I think there's a few cases of that

Reply:
Good idea. Fixed.